.travis.yml

```diff
@@ -2,25 +2,38 @@ language: csharp
 solution: JSIL.sln
 
 before_install:
-  - wget -P /tmp/avim -N https://ftp.mozilla.org/pub/mozilla.org/firefox/nightly/latest-mozilla-central/jsshell-linux-x86_64.zip
-  - unzip -d /tmp/js /tmp/avim/jsshell-linux-x86_64.zip
+  - wget -P /tmp/ -N https://ftp.mozilla.org/pub/mozilla.org/firefox/nightly/latest-mozilla-central/jsshell-linux-x86_64.zip
+  - unzip -d /tmp/js /tmp/jsshell-linux-x86_64.zip
   - export PATH=$PATH:/tmp/js
 
 install:
   - export NODEJS=$(dirname `nvm which $NODEJS_VERSION`)
   - export FSHARPINSTALLDIR=$PWD/packages/FSharp.Compiler.Tools.4.0.0.1/tools/
-  - ci/download-asset.sh packages .
   - nuget restore JSIL.sln
   - nuget install NUnit.Runners -Version 2.6.4 -OutputDirectory packages 
   - nuget install FSharp.Compiler.Tools -Version 4.0.0.1 -OutputDirectory packages 
-  - ci/upload-asset.sh packages packages &
+
+before_script:
+  - mv /tmp/JSIL_Tests "/tmp/JSIL Tests" || true
```

Comment:
Main benefit here, is that cache accessible (I hope read-only, but have not tested) with pull-request builds, so such builds should also run faster.